lib/package.rb

```diff
@@ -157,6 +159,14 @@ def self.#{prop}
     end
   end
 
+  def self.compatible?
+    if @compatibility
+      return (@compatibility.casecmp?('all') or @compatibility.include?(ARCH))
+    else
+      abort "#{name}: Missing `compatibility` field.".lightred
```

Comment:
Return `false` would be better since:
```
abort "Package #{@pkg.name} is not compatible with your device architecture (#{ARCH}) :/".lightred unless @pkg.compatible?
```
?

Reply:
Change made